### PR TITLE
main-page updates

### PR DESCRIPTION
**Pull request by jdville03**
This is a PR to merge into `main-page` branch before that gets merged into `master`.

@sophiacodes apologies for the way I'm giving comments on your original PR. I wasn't super familiar with this codebase so wanted to validate my changes live as I went through. ~The changes look worse than they actually are because I ran prettier for one of the last commits.~

Substantively, I fixed a few links (some dead and others pointing to wrong targets), cleaned up some typos, and did some light editing on content to make it consistent with the other content that is on the static site as well as with how we are describing some of our products in the app.

I think the only thing left to do would be removing the newsletter signup form in the footer. I think @jtrein mentioned he would tackle that.

Let me know if you have any questions about the changes.

### CODE REVIEW

File: src/components/SignupForm.jsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ export default class SignupForm extends Component {
     return (
       <div className={styles.signup}>
         <div>
-          <H1>Build borderless commericial systems on the blockchain.</H1>
+          <H1>Build borderless commericial systems on the blockchain</H1>
```
Comment (outkaj) [inline src/components/SignupForm.jsx]: caught typo in commercial here

Comment (jdville03) [reply src/components/SignupForm.jsx]: Fixed now. Thanks! The more eyes the better on things like this!

**Comment by jdville03** [comment]
You can disregard my initial comment earlier about the changes due to running prettier. I force pushed without those because they were so not helpful for you to see the actual changes.

**Review by sophiacodes — APPROVED** [review]
@jdville03 thank you for scrutinizing this! These changes look great!

**Pull request merged**